src/administration/integrations/gluescripts.md

```diff
@@ -0,0 +1,248 @@
+
+Create a new Javascript file inside the `scripts` folder with the following content:
+
+```javascript
+{
+  "type": "script",
+  "script": "<CODE>"
+}
+```
+
+where:
+* `CODE` is the Javascript code of the integration
+
+#### Custom Slack integration
+
+For example, the GlueScript can perform a custom Slack integration as follow:
+
+```javascript
+var message = activity.text;
+var color;
+
+if (activity.state == 'complete') {
+  if (activity.result == 'success') {
+    color = '#66c000';
+  } else {
+    color = '#ff0000';
+  }
+}
+
+var resp = fetch(
+  '<SLACK_PAYLOAD_URL>',
+  {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      'attachments': [{
+        'fallback': message,
+        'color': color,
+        'fields': [{
+          'value': message,
+        }],
+      }],
+    }),
+  }
+);
+
+if (!resp.ok) { throw 'Failed'; }
+```
+
+where:
+* `SLACK_PAYLOAD_URL` is the URL provided by Slack for your custom webhook
+
+### 2. Commit the script
+
+Commit the script to Git:
+
+```bash
+git add .platform/scripts/SCRIPT.js
+git commit -m "Committing the GlueScript file."
+git push platform master
+```
+
+where:
+* `SCRIPT` is the name of your Javascript file
+
+### 3. Enable the integration
+
+Enable the GlueScript integration as follows (you need to have the Platform.sh CLI installed):
+
+```bash
+platform integration:add --type script --file .platform/scripts/SCRIPT.js
+```
+
+where:
+* `SCRIPT` is the name of your Javascript file
+
+> **note**
+>
+> The GlueScript integration can only be managed by the project admin.
+
+# GlueScript APIs
+
+## Project API
+
+The project API gives you everything you need to know about the project in the context of which the Gluescript is running. 
+
+For example: `project.environments.master` will give you access to the metadata of the master environment of the current project.
+
+The API starts at `api.project` and is based on a one resource == one object model.
+
+### Properties
+
+The properties of the resource can be access directly on the object, like:
+
+```javascript
+project.owner
+```
+
+### Sub-objects
+
+Sub-objects can be accessed directly on the object, like:
+
+```javascript
+project.settings
+```
+
+### Collections
+
+Resources that are collections support iteration:
+
+```javascript
+project.environments.forEach(function(env) {
+  console.log(env.name);
+});
+```
+
+They also support direct access to an element:
+
+```javascript
+project.environments["master"].name
+```
+
+### Actions
+
+Resource actions are accessible as methods on the object:
+
+```javascript
+project.environments["master"].branch({
+  title="Test",
+  name="test",
+})
+```
+
+## Activity API
+
+The activity API accessed through `activity` gives you all the metadata about the current activity that invoked the Gluescript. The schema is the same as the one for Webhooks, documented here: https://docs.platform.sh/administration/integrations/webhooks.html
```

Comment:
Remove the raw link:

The scheme is the same as the one for [Webhooks](https://docs.platform.sh/administration/integrations/webhooks.html).